src/components/Toast/types.tsx

```diff
@@ -8,6 +8,7 @@ export type ToastProps = {
     icon: ReactNode;
     animationDirection?: ToastAnimationDirection;
     children?: ReactNode;
+    variant?: ToastVariant;
```

Comment:
Thanks a lot! @fulopdaniel 😄 
It looks good to me, the only request I'm going to make before we merge this as I just aligned with designers...is renaming the property "variant" to "theme". We plan to use `variant` in other components for something different. 😉